.chloggen/elasticsearch-flush-time.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Since we were already collecting this but not emitting it, and because there is already a metric for this in the data model, I think this should be considered a bug fix.

Reply:
It was omitted on purpose by me - I reused the `operation` attribute when adding `search` metrics, so it's natural that all unused operations type are collected, but not emitted.

Reply:
I see, but this PR does not add a new metric. It adds a new data point to an existing metric. The result is that the existing metric will have a different "total" than before. 

If we think adding this new data point gives us an accurate total, then this is a bug fix. Otherwise, if we think the "total" is still only part of the picture, then we could look to complete the total by adding additional data points.

Reply:
I see, but this PR does not add a new metric. It adds a new data point to an existing metric. The result is that the existing metric will have a different "total" than before. 

If we think adding this new data point gives us an accurate total, then this is a bug fix. Otherwise, if we think the "total" is still only part of the picture, then we could look to complete the total by adding additional data points.

Reply:
To sum up: do you think that we should add all missing data points at once?
For example, https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14871 is a very similar pull request to this one. If we want to add missing data points at once, I will close both PRs and send another one, that adds all missing data points.

Reply:
If there are multiple missing data points for the same metric, I think it's appropriate to add them in one PR.